code/modules/projectiles/projectile.dm

```diff
@@ -61,6 +61,8 @@
 	var/list/atom/movable/uncross_scheduled = list() // List of border movable atoms to check for when exiting a turf.
 
 	var/damage = 0
+	var/penetration = 0
+	var/sundering = 0
```

Comment:
New vars must be autodoced

/// A comment on what is penetration
var/penetration = 0